src/main/resources/application.yml

```
@@ -17,10 +17,25 @@ spring:
+    web:
+      exposure:
+        include: health,info
 
 auth:
   jwt:
     secret: ${JWT_SECRET_KEY}
   kakao:
     rest-api-key: ${KAKAO_REST_API_KEY}
-    redirect-uri: http://localhost:8080/auth/kakao/login
\ No newline at end of file
+    redirect-uri: http://localhost:8080/auth/kakao/login/callback
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)